### DISCUSSION TITLE
ScriptReference/MonoBehaviour.OnTriggerExit2D

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#636] [Notes]
# ScriptReference/MonoBehaviour.OnTriggerExit2D

https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnTriggerExit2D.html

**Comment by qwe321qwe321qwe321** [comment]
[OnTriggerExit2D doesn't work by changing localScale to a tiny number.](https://forum.unity.com/threads/ontriggerexit2d-doesnt-work-by-changing-localscale-to-a-tiny-number.1092973/)
